neural_tangents/stax.py

```diff
@@ -91,7 +86,13 @@
 from neural_tangents.utils.typing import AnalyticKernelFn, Axes, Get, InitFn, InternalLayer, Kernels, Layer, LayerKernelFn, PyTree, Shapes
 import scipy as osp
 
-
+import tensorflow as tf
```

Comment:
Eventually, of course, we'd have to think of a nicer way to do this.

Reply:
Sorry, Sam, a nicer way to ... ?